wagtailmodelchoosers/client/decorators/GenericModelDecorator.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const GenericModelEntity = (props) => {
+  console.log(props);
+  return React.createElement(
```

Comment:
If you're not using the version of `React` directly from Wagtail 2.0 at `window.React`, then this should be written with JSX.